tests/flow/test_traversal_construction.py

```diff
@@ -0,0 +1,75 @@
+import os
+from RLTest import Env
+from redisgraph import Graph
+
+graph = None
+
+class testTraversalConstruction():
+    def __init__(self):
+        global graph
+
+        self.env = Env(decodeResponses=True)
+        redis_con = self.env.getConnection()
+        graph = Graph("TraversalConstruction", redis_con)
+
+    # make sure traversal begins with labeled entity
+    def test_start_with_label(self):
+        queries = ["MATCH (A:L)-->(B)-->(C) RETURN 1",
+                   # "MATCH (A)-->(B:L)-->(C) RETURN 1", # improve on this case
```

Comment:
This case still fails.

Reply:
Thanks for the explanation on this case, I had missed that nuance! The behavior here makes sense.